.github/CODEOWNERS

```diff
@@ -0,0 +1,29 @@
+# See https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# Global owners, default owners for everything in the repo
+*                                               @swissspidy
+
+# All server-side logic.
+*.php                                           @swissspidy @spacedmonkey @miina
+composer.json                                   @swissspidy @spacedmonkey @miina
+phpcs.xml                                       @swissspidy @spacedmonkey @miina
+phpstan.neon.dist                               @swissspidy @spacedmonkey @miina
+phpunit.xml.dist                                @swissspidy @spacedmonkey @miina
+
```

Comment:
Should we set @miina as owner of movable?

Reply:
Thoughts, @miina?

Since Movable usage is split across a few individual files and folders, we'd just have to make sure the list here in this file is updated accordingly when refactoring etc.

Reply:
Thanks for tagging this, @spacedmonkey 

Ideally yes, would be great.
Q: We still follow the idea of requesting a review from everyone in case of JS-related PR-s, correct?
If yes, then it's fine if some file is missed for the automatic review request with refactoring later, it's OK to add just the main movable files for now (in the `movable` folder + the 2 files in the `canvas` folder: `multiSelection-` and `singleSelectionMovable`) .

Reply:
This change would just automatically add code owners as reviewers, but of course one can & should still add more reviewers.

In the future, we could think about requiring reviews from code owners.